securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def logger_setup(verbose=False):
+    """ Configure logging handler """
+    # Set default level on parent
+    sdlog.setLevel(logging.DEBUG)
+    level = logging.DEBUG if verbose else logging.INFO
+
+    stdout = logging.StreamHandler(sys.stdout)
```

Comment:
> I mentioned that you could send logger output to sys.stdout earlier, but that's not necessarily the best place to send it.

I have to push back here buddy. I've already made a number of changes to the output at your request. You already brought up a nit that I should be utilizing the `logger` module to stdout -- okay, so I added that. When you change your mind after I spend the time to address the original comment -- it makes for an extremely annoying peer review experience. At this point, if I make more changes I'm not convinced you wont have more concerns about implementation after I write the code and complete all the testing.

I have to insist at this point that you either modify the script to meet your specifications and push to this PR (make sure you test in `Tails` first) or give me a more succinct list of exactly what you have in mind and we can come to a compromise.